### DISCUSSION TITLE
how to change the paste behavior?

### DISCUSSION THREAD

**Discussion by PorkDumplings** [Milkdown/milkdown#481] [Q&A] [answered]
I just found a problem use milk down.
when i copy the text like this:
```markdown
[style](url)  **block** *iltiac*
```
then i paste it  insert one paragraph in milkdown , i found the insert text at new one paragraph . here is screenshots
before paste
![image](https://user-images.githubusercontent.com/50495414/166861782-bac72871-3070-43f9-97d3-4f5483cddffa.png)
after paste use that text
![image](https://user-images.githubusercontent.com/50495414/166861914-81a33fa9-88c4-4dc7-9d00-b16bb3e4aae1.png)

i want paste behavior the text like this no paragraph break
![image](https://user-images.githubusercontent.com/50495414/166862037-e8751541-f449-4ba2-8815-188de8e6f012.png)

any idea to change this ? thanks

**Comment by Saul-Mirone** [comment, marked as answer]
I think there's no way to change the behavior now. And IMO it's a bug, I'll fix it.

**Reply by PorkDumplings** [reply]
ok  thanks for reply
